src/neo/Trie/MPT/MPTNode/ShortNode.cs

```diff
@@ -0,0 +1,48 @@
+using System.IO;
+using Neo.Cryptography;
+using Neo.IO;
+
+namespace Neo.Trie.MPT
+{
+    public class ShortNode : MPTNode
+    {
+        public byte[] Key;
+
+        public MPTNode Next;
+
+        public new int Size => Key.Length + Next.Size;
+
+        protected override byte[] calHash()
+        {
+            return Key.Concat(Next.GetHash()).Sha256();
+        }
+        public ShortNode()
+        {
+            nType = NodeType.ShortNode;
+        }
+
+        public ShortNode Clone()
+        {
+            var cloned = new ShortNode()
+            {
+                Key = (byte[])Key.Clone(),
+                Next = Next,
+            };
+            return cloned;
+        }
+
+        public override void Serialize(BinaryWriter writer)
+        {
+            base.Serialize(writer);
+            writer.WriteVarBytes(Key);
+            writer.WriteVarBytes(Next.GetHash());
+        }
+
+        public override void Deserialize(BinaryReader reader)
+        {
+            Key = reader.ReadVarBytes();
+            var hashNode = new HashNode(reader.ReadVarBytes());
+            Next = hashNode;
+        }
+    }
+}
```

Comment:
A blank line should be added in the end of the file, otherwise the dotnet-format won't pass.